diff --git docs/installation.rst docs/installation.rst
index c3d3941..bde880c 100644
--- docs/installation.rst
+++ docs/installation.rst
@@ -202,6 +202,11 @@ Build Options
   the libraries are not found. Webpmux (WebP metadata) relies on WebP
   support. Tcl and Tk also must be used together.
 
+* Build flag: ``--disable-osx-tcltk-framework``. Skips linking against
+  the OSX system TCL/TK frameworks for systems that have alternate
+  versions of the libraries installed. 
+
+
 Sample Usage::
 
     $ MAX_CONCURRENCY=1 python setup.py build_ext --enable-[feature] install
diff --git setup.py setup.py
index 0f74a40..873459c 100644
--- setup.py
+++ setup.py
@@ -121,9 +121,13 @@ class pil_build_ext(build_ext):
         ('disable-%s' % x, None, 'Disable support for %s' % x) for x in feature
     ] + [
         ('enable-%s' % x, None, 'Enable support for %s' % x) for x in feature
+    ] + [
+        ('disable-osx-tcltk-framework', None,
+             'Disable linking against system tcl/tk frameworks on OSX')
     ]
 
     def initialize_options(self):
+        self.disable_osx_tcltk_framework = None
         build_ext.initialize_options(self)
         for x in self.feature:
             setattr(self, 'disable_%s' % x, None)
@@ -580,7 +584,7 @@ class pil_build_ext(build_ext):
                                   define_macros=defs))
 
         if feature.tcl and feature.tk:
-            if sys.platform == "darwin":
+            if sys.platform == "darwin" and not self.disable_osx_tcltk_framework:
                 # locate Tcl/Tk frameworks
                 frameworks = []
                 framework_roots = [
