--- Pillow-11.0.0.orig/setup.py	2024-10-15 07:55:00.000000000 +0200
+++ Pillow-11.0.0/setup.py	2024-10-16 08:56:01.786769874 +0200
@@ -854,12 +854,6 @@
         if feature.get("tiff"):
             libs.append(feature.get("tiff"))
             defs.append(("HAVE_LIBTIFF", None))
-            if sys.platform == "win32":
-                # This define needs to be defined if-and-only-if it was defined
-                # when compiling LibTIFF. LibTIFF doesn't expose it in `tiffconf.h`,
-                # so we have to guess; by default it is defined in all Windows builds.
-                # See #4237, #5243, #5359 for more information.
-                defs.append(("USE_WIN32_FILEIO", None))
         if feature.get("jpeg"):
             libs.append(feature.get("jpeg"))
             defs.append(("HAVE_LIBJPEG", None))
