From c60bdf31dade079f53c217edd92230566e700db4 Mon Sep 17 00:00:00 2001
From: Andrew Murray <radarhere@users.noreply.github.com>
Date: Thu, 11 Nov 2021 20:47:46 +1100
Subject: [PATCH 2/3] Only prefer command line tools SDK on macOS over the
 default

---
 setup.py | 28 ++++++++++++++++++----------
 1 file changed, 18 insertions(+), 10 deletions(-)

diff --git a/setup.py b/setup.py
index 10c1b585..341cb331 100755
--- a/setup.py
+++ b/setup.py
@@ -533,16 +533,24 @@ class pil_build_ext(build_ext):
                 _add_directory(include_dirs, "/usr/X11/include")
 
             # SDK install path
-            sdk_path = "/Library/Developer/CommandLineTools/SDKs/MacOSX.sdk"
-            if not os.path.exists(sdk_path):
-                try:
-                    sdk_path = (
-                        subprocess.check_output(["xcrun", "--show-sdk-path"])
-                        .strip()
-                        .decode("latin1")
-                    )
-                except Exception:
-                    sdk_path = None
+            try:
+                sdk_path = (
+                    subprocess.check_output(["xcrun", "--show-sdk-path"])
+                    .strip()
+                    .decode("latin1")
+                )
+            except Exception:
+                sdk_path = None
+            if (
+                not sdk_path
+                or sdk_path == "/Applications/Xcode.app/Contents/Developer"
+                "/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk"
+            ):
+                commandlinetools_sdk_path = (
+                    "/Library/Developer/CommandLineTools/SDKs/MacOSX.sdk"
+                )
+                if os.path.exists(commandlinetools_sdk_path):
+                    sdk_path = commandlinetools_sdk_path
             if sdk_path:
                 _add_directory(library_dirs, os.path.join(sdk_path, "usr", "lib"))
                 _add_directory(include_dirs, os.path.join(sdk_path, "usr", "include"))
-- 
2.32.0.windows.2

