From 17a5ec5a144ae824443ca56d2787c43e81513920 Mon Sep 17 00:00:00 2001
From: Andrew Murray <radarhere@users.noreply.github.com>
Date: Wed, 12 Jan 2022 11:35:32 +1100
Subject: [PATCH 2/3] Added patch to fix failing Windows test

---
 src/PIL/TiffImagePlugin.py | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/src/PIL/TiffImagePlugin.py b/src/PIL/TiffImagePlugin.py
index de2ce066e..7c8d1f5fc 100644
--- a/src/PIL/TiffImagePlugin.py
+++ b/src/PIL/TiffImagePlugin.py
@@ -1346,6 +1346,10 @@ class TiffImageFile(ImageFile.ImageFile):
         # file descriptor, use that instead of reading
         # into a string in python.
         assert self.fp is not None
+        position = self.fp.tell()
+        self.fp.seek(0)
+        fp = io.BytesIO(self.fp.read())
+        self.fp.seek(position)
         try:
             fp = hasattr(self.fp, "fileno") and self.fp.fileno()
             # flush the file descriptor, prevents error on pypy 2.4+
